Frontend/ui-customer/src/Screens/ZooTicketBookingFlow/TicketBookingScreen/TicketInfoScreen/Controller.purs

```diff
@@ -71,11 +75,18 @@ eval (TicketQRRendered id text) state  =
   ]
 
 eval (ShareTicketQR serviceName) state = do
-  _ <- pure $ shareImageMessage "Embark on a wild adventure at Alipore Zoo! 🐅🌿 Your tickets are ready to unlock a day of fun and discovery. See you soon! 🎟️👀 #AliporeZooAdventures" (shareImageMessageConfig serviceName)
+  let textMessage = getTextMessage state.data.selectedBookingInfo.ticketPlaceName
+  _ <- pure $ shareImageMessage textMessage (shareImageMessageConfig serviceName)
   continue state
 
 eval GoHome state = exit GoToHomeScreen
 
+eval (Copy text) state = continueWithCmd state [ do 
+    _ <- pure $ copyToClipboard text
+    _ <- pure $ toast (getString COPIED)
```

Comment:
This will look cleaner
`void $ pure $ toast $ getString COPIED`

Reply:
done